tests/src/test/scala/slinky/core/TagTest.scala

```diff
@@ -57,7 +56,7 @@ class TagTest extends FunSuite {
   }
 
   test("Mouse events can be given a function taking a SyntheticMouseEvent") {
-    assertCompiles("div(onMouseOver := (v => { (v: SyntheticMouseEvent[Element]) }))")
+    div(onMouseOver := ((_: SyntheticMouseEvent[TagElement#RefType]) => ()))
```

Comment:
Ideally this would keep the same form as the original code, the intended purpose of the more complex definition was to make sure that Scala's type inference could handle the API.